### PR TITLE
Reject queries with multiple patterns

### LINKED ISSUE #62: Panic with invalid .tsg
The following .tsg input parses, but can lead to a panic at run time:

```
(const_item)
(function_item)
{}
```

**To reproduce:**

```
git clone git@github.com:jorendorff/tree-sitter-rust.git
cd tree-sitter-rust
script/setup
script/test
```

**Backtrace:**

```
thread 'main' panicked at 'index out of bounds: the len is 1 but the index is 1', /home/jorendorff/.cargo/registry/src/github.com-1ecc6299db9ec823/tree-sitter-graph-0.5.0/src/lazy_execution.rs:62:27
stack backtrace:
   0: rust_begin_unwind
             at /rustc/db9d1b20bba1968c1ec1fc49616d4742c1725b4b/library/std/src/panicking.rs:498:5
   1: core::panicking::panic_fmt
             at /rustc/db9d1b20bba1968c1ec1fc49616d4742c1725b4b/library/core/src/panicking.rs:107:14
   2: core::panicking::panic_bounds_check
             at /rustc/db9d1b20bba1968c1ec1fc49616d4742c1725b4b/library/core/src/panicking.rs:75:5
   3: tree_sitter_graph::execution::<impl tree_sitter_graph::ast::File>::execute_into
   4: tree_sitter_stack_graphs::StackGraphLanguage::build_stack_graph_into
   5: tree_sitter_stack_graphs::test::Command::run_test_with_context
   6: tree_sitter_stack_graphs::test::Command::run
   7: tree_sitter_stack_graphs::main
```

### PR DESCRIPTION
**Pull request by hendrikvanantwerpen**
Add check that a query contains only one pattern.

Fixes #62.

### CODE REVIEW

**Review by tausbn — APPROVED** [review]
Looks good to me. :+1:

File: src/parser.rs [hunk 1]
```diff
@@ -72,6 +70,8 @@ pub enum ParseError {
     UnexpectedKeyword(String, Location),
     #[error("Unexpected literal '#{0}' at {1}")]
     UnexpectedLiteral(String, Location),
+    #[error("Query contains multiple patterns {0}")]
```
Comment (dcreager) [inline src/parser.rs:73]: nit: should this be `at {0}`?

File: src/parser.rs [hunk 2]
```diff
@@ -60,8 +60,6 @@ pub enum ParseError {
     InvalidRegex(String, Location),
     #[error("Expected integer constant in regex capture at {0}")]
     InvalidRegexCapture(Location),
-    // TODO: The positions in the wrapped QueryError will be incorrect, since they will count the
-    // row/column from the start of the query, not from the start of the file.
```
Comment (dcreager) [inline src/parser.rs:64]: This is removed but I don't a corresponding code change. Was it fixed earlier?

**Review by dcreager — APPROVED** [review]

File: src/parser.rs (hunk 2 above)
Comment (hendrikvanantwerpen) [reply src/parser.rs:64]: Yes, it was fixed in an [earlier commit](https://github.com/tree-sitter/tree-sitter-graph/commit/f80d5a2cba316a09aa08425a78a3e08d4d645817) but I overlooked this comment then.

File: src/parser.rs (hunk 1 above)
Comment (hendrikvanantwerpen) [reply src/parser.rs:73]: It should!

**Pull request merged**